test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -39,10 +29,21 @@
 
 import javax.swing.JFileChooser;
 import javax.swing.JFrame;
+import javax.swing.JOptionPane;
 import javax.swing.SwingUtilities;
 import javax.swing.WindowConstants;
 
-public class FileChooserSymLinkTest{
+/** @test
+  * @bug 8281966
```

Comment:
```suggestion
/*
 * @test
 * @bug 8281966
```

It is not a javadoc comment, only one asterisk should be used. Please align the following lines by removing one space.